transform/mattermost-analytics/models/intermediate/data_eng/signup/int_signups_aggregated_to_users.sql

```diff
@@ -0,0 +1,45 @@
+{{
+    config({
+        "materialized": "table",
+        "incremental_strategy": "merge",
+        "unique_key": ['portal_customer_id'],
+        "merge_update_columns": ['verify_email']
+    })
+}}
+
+WITH identifies as (
+    SELECT user_id, 
+        coalesce(portal_customer_id, context_traits_portal_customer_id) as portal_customer_id,
+        ROW_NUMBER() OVER (PARTITION BY user_id ORDER BY RECEIVED_AT) AS row_number
```

Comment:
Why not use `qualify` in identifies CTE?

If I understand correctly, the idea is to create a mapping between `user_id` and `portal_customer_id`. For this case, the logic is rather simple, as for each `user_id` a single `portal_customer_id` must exist and vice versa. This can be moved to a separate intermediate model because:
-  it's extremely likely that it will be reused, 
- this way tests on the 1:1 mapping can be added.

In fact it's a common challenge when using data from CDP platforms. Here's a few examples on how it's performed using DBT:
- [Rudderstack](https://github.com/rudderlabs/dbt-sessionization/blob/1.0.1/models/rudder/dbt_aliases_mapping.sql)
- [Segment](https://github.com/dbt-labs/segment/blob/main/models/sessionization/segment_web_user_stitching.sql)